qulice-checkstyle/src/main/java/com/qulice/checkstyle/LineRange.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2011-2016, Qulice.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the Qulice.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.qulice.checkstyle;
+
+/**
+ * Represent a line range. For example, a Java method can be described by an
+ * instance of this class. The alpha line could be the method definition and
+ * the omega line could be the end closing bracket.
+ *
+ * @author Jimmy Spivey (JimDeanSpivey@gmail.com)
+ * @version $Id$
+ */
+public final class LineRange {
+
+    /**
+     * The first (alpha) line number in the range.
+     */
+    private final transient int alpha;
```

Comment:
I'm forced to create doppleganger variable names because the constructor won't allow this:

```java
    public LineRange(final int first, final int last) {
        this.first = first;
        this.first = last;
    }
```

Also, due to qulice conventions, I am not able to have these getter names, as instance variables cannot have the same name as method names:

```java
    public int first() {
        return this.alpha;
    }

    public int last() {
        return this.omega;
    }
```

I am under the impression that it is better to name the parameters to the constructor first/last, and the getters named first/last and let the encapsulated fields have the more unusual name.